src/index.ts

```diff
@@ -64,104 +156,192 @@ const dmMachine = createMachine(
             states: {
               Greeting: {
                 entry: "speak.greeting",
-                on: { SPEAK_COMPLETE: "HowCanIHelp" },
+                on: { SPEAK_COMPLETE: "help" },
               },
-              HowCanIHelp: {
-                entry: say("You can say whatever you like."),
+              help: {
+                entry: say("Which music would you like to play?"),
                 on: { SPEAK_COMPLETE: "Ask" },
               },
               Ask: {
                 entry: listen(),
                 on: {
                   RECOGNISED: {
-                    target: "Repeat",
+                    target: "AskChatGPT",
                     actions: [
-                      ({ event }) => console.log(event),
                       assign({
                         lastResult: ({ event }) => event.value,
                       }),
                     ],
-                  },
+                    target: 'CheckSlots'
```

Comment:
You should only have one target for the transition (or one for one guard if you have guards)